src/main/java/HTTP/HTTPToken.java

```diff
@@ -0,0 +1,45 @@
+package HTTP;
+import lombok.SneakyThrows;
+import org.apache.http.impl.client.CloseableHttpClient;
+
+import static HTTP.HTTPVariables.URL_FOR_READ_TOKEN;
+import static HTTP.HTTPVariables.URL_FOR_WRITE_TOKEN;
+
+public class HTTPToken {
+
+    private static String writeToken;
+    private static String readToken;
+
+    HTTPBasicAuthClient httpBasicAuthClient = new HTTPBasicAuthClient();
+
+    @SneakyThrows
+    private String getOauthToken(CloseableHttpClient httpClient, String URL) {
+        HTTPResponseDTO httpResponseDTO = httpBasicAuthClient.callPostReturnHTTPResponseData(httpClient, URL);
+        return httpResponseDTO.getBody().getString("access_token");
+    }
```

Comment:
Shouldn't be there. Please move it to the HttpClient class

Reply:
https://github.com/artsiomMandrukevich/coherent-rest-api-testing/pull/1#discussion_r1231329504

Reply:
done